src/main/java/greymerk/roguelike/config/RogueConfig.java

```diff
@@ -23,7 +23,8 @@ public enum RogueConfig {
     DONOVELTYSPAWN,
     UPPERLIMIT,
     LOWERLIMIT,
-    ROGUESPAWNERS;
+    ROGUESPAWNERS,
+    LEGACYNBT;
```

Comment:
Rather than a config, could this check the json element read in to determine which format it is in? They seem to me to be very structurally different which should make this more possible, though let me know if I'm missing anything

Reply:
It's the sensible thing to do, done.